mix.exs

```diff
@@ -44,8 +44,9 @@ defmodule MvOpentelemetry.MixProject do
     [
       # OpenTelemetry
       {:opentelemetry_telemetry, "~> 1.0"},
-      {:opentelemetry_api, "~> 1.0", override: true},
```

Comment:
This override is no longer needed, from the look of it. It was originally in place to allow for using telemetry 0.4 but this is no longer supported.